gensim/utils.py

```diff
@@ -1164,11 +1164,12 @@ def check_output(stdout=subprocess.PIPE, *popenargs, **kwargs):
     Added extra KeyboardInterrupt handling
     """
     try:
+        cmd = kwargs.get("args")
+        logger.info("COMMAND: %s", cmd)
```

Comment:
Made suggested changes, and removed logging command(`cmd`) at `logger.info` level